datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/readmes.py

```diff
@@ -141,9 +141,6 @@ def validate_readme(integration, repo, display_queue, files_failed, readme_count
 def get_ascii_enforcement_error_lines(contents):
     errors_lines = []
     for i, line in enumerate(contents.splitlines()):
-        # Ignore new line
-        line = line[:-1]
```

Comment:
This is to avoid printing new lines; at least put it behind a `if lines[-1] == '\n'`

Reply:
sure!

Reply:
or, `line = line.rstrip('\n')`

Reply:
Updated!